app/config.py

```diff
@@ -6,6 +6,7 @@
 
 basedir = os.path.abspath(os.path.dirname(__file__))
 configdir = os.path.join(basedir, 'config')
```

Comment:
Can we get rid of this now, and just construct the basecfg & override dirs off of `basedir`?